CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+## 2.11.0-beta.1
+
+### Features ✨ and improvements 🏁
+
+* Improve symbol placement performance with globe projection ([#12105](https://github.com/mapbox/mapbox-gl-js/pull/12105))
```

Comment:
Nit: Should this go at the bottom of this section? IMO the bounds support might be the most impactful parts of this release and it would be best to list them first.